{"uuid": "6e9ab2c2-1bb2-456c-abf1-c23b577ac41b", "children": ["01f0be6d-19a0-440d-abb3-eba299ebf5dc"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x560c6a569ffa <unknown>\n#1 0x560c6a028970 <unknown>\n#2 0x560c6a07a385 <unknown>\n#3 0x560c6a07a5b1 <unknown>\n#4 0x560c6a0c93c4 <unknown>\n#5 0x560c6a0a02bd <unknown>\n#6 0x560c6a0c670c <unknown>\n#7 0x560c6a0a0063 <unknown>\n#8 0x560c6a06c328 <unknown>\n#9 0x560c6a06d491 <unknown>\n#10 0x560c6a53142b <unknown>\n#11 0x560c6a5352ec <unknown>\n#12 0x560c6a518a22 <unknown>\n#13 0x560c6a535e64 <unknown>\n#14 0x560c6a4fcbef <unknown>\n#15 0x560c6a558558 <unknown>\n#16 0x560c6a558736 <unknown>\n#17 0x560c6a568e76 <unknown>\n#18 0x7fac6ae9caa4 <unknown>\n#19 0x7fac6af29c3c <unknown>\n\n", "trace": "  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/src/tests/ui_tests/conftest.py\", line 30, in driver\n    WebDriverWait(driver, 10).until(\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1743006769158, "stop": 1743006780994}], "start": 1743006769158, "stop": 1743006781001}